### PR TITLE
Added functions that are creation and removing a section using prompt widget.

### PR DESCRIPTION
**Pull request by himaaaatti**

### CODE REVIEW

**Comment by himaaaatti** [comment]
This PR work like this gif.
output

File: libqtile/layout/tree.py [hunk 1]
```diff
@@ -637,6 +642,30 @@ def cmd_decrease_ratio(self):
         self.panel_width -= 10
         self.group.layoutAll()
 
+    def cmd_add_section_using_prompt(self):
+        prompt = self.group.qtile.widgetMap['prompt']
```
Comment (flacjacket) [inline libqtile/layout/tree.py]: I think it would be a good idea to parameterize the widget name here.

File: libqtile/widget/prompt.py [hunk 2]
```diff
@@ -434,11 +434,15 @@ def startInput(self, prompt, callback,
         self.show_cursor = self.cursor
         self.cursor_position = 0
         self.callback = callback
-        self.completer = self.completers[complete](self.qtile)
+        if not complete:
```
Comment (flacjacket) [inline libqtile/widget/prompt.py]: Why is this needed? Can you not use the `NullCompleter`?

**Comment by himaaaatti** [comment]
merged a head of develop branch.

**Comment by himaaaatti** [comment]
I'll consider about more generalized method.
